Tools/TestWebKitAPI/Tests/WebCore/ApplicationManifestParser.cpp

```diff
@@ -444,6 +463,23 @@ TEST_F(ApplicationManifestParserTest, StartURL)
     testStartURL("\"../page2\""_s, "https://example.com/page2"_s);
 }
 
+TEST_F(ApplicationManifestParserTest, Dir)
+{
+    testDir("123"_s, ApplicationManifest::Direction::Auto);
```

Comment:
Can put `using enum ApplicationManifest::Direction` at the top of the function and make the entire thing more terse without losing clarity.